stable/nginx-ingress/README.md

```diff
@@ -52,6 +52,7 @@ Parameter | Description | Default
 `controller.config` | nginx ConfigMap entries | none
 `controller.hostNetwork` | If the nginx deployment / daemonset should run on the host's network namespace | false
 `controller.defaultBackendService` | default 404 backend service; required only if `defaultBackend.enabled = false` | `""`
+`controller.defaultSSLCertificate` | Name of the secret that contains a SSL certificate to be used as default for a HTTPS catch-all server | `""`
```

Comment:
One more nit...
> a SSL
should be:
> an SSL

and
> a HTTPS
should be:
> an HTTPS

If someone wants to dig into the language it depends on how you pronounce them. There [a stackexchange post on it](https://english.stackexchange.com/questions/1016/do-you-use-a-or-an-before-acronyms).

Reply:
@mattfarina haha, fair point. I just copied it from the upstream code that you mentioned before :)

https://github.com/kubernetes/ingress-nginx/blob/908807a76d42694b9abddd62b809d31490e222e0/pkg/ingress/controller/launch.go#L80-L81